build.gradle

```diff
@@ -167,8 +167,10 @@ allprojects {
           // ZooKeeper (potentially older and containing CVEs)
           libs.nettyHandler,
           libs.nettyTransportNativeEpoll,
-          // be explicit about the reload4j version instead of relying on the transitive versions
-          libs.reload4j
+          libs.reload4j,
```

Comment:
We still need reload4j here?

Reply:
It looks like it's our test depends on reload4j. In this case, we should try to fix it.

Reply:
I see. 
I will try to deal with this issue.